docs/development/l10n.rst

```diff
@@ -159,15 +159,22 @@ HTML content
 ^^^^^^^^^^^^
 
 Some source strings represent HTML that will be presented in the
-SecureDrop web interface. It can be hard to tell what to translate,
-since HTML is source code and changing the wrong thing can break the
-page layout.
+SecureDrop web interface.
 
-One thing you should always translate are ``alt`` attributes of image
-elements. Image elements (``<img>``) in HTML place a picture on the
+HTML elements (embraced by in ``<``, ``>``, example: ``<strong>``)
+can contain multiple so-called *attributes*.
+
+The text of the two attributes called ``alt`` and ``title``
```

Comment:
As a general note, I would suggest changing phrases like  "the text of the attribute X" to "the text content of the attribute X", to make it even clearer that we are not talking about the attribute name itself.